src/restapi/addons/get.test.tsx

```diff
@@ -0,0 +1,34 @@
+import { renderHook, waitFor } from '@testing-library/react';
```

Comment:
For some reason, old things from the other PR got in here... Maybe it needs a merge/rebase.

Reply:
@sneridagh I rebased this PR onto main. It should be good to go now.